server/tests/utils/comment.test.js

```diff
@@ -0,0 +1,29 @@
+import { expect } from 'chai';
+import { describe } from 'mocha';
```

Comment:
No need to `import { describe }  from mocha`
You can use it without importing it